lib/helpers/utils.js

```diff
@@ -1558,7 +1553,7 @@ export async function parsePkgLock(pkgLockFile, options = {}) {
         continue;
       }
       const depPurlString = decodeURIComponent(
```

Comment:
I think we can remove decodeURIComponent and the .replace here.

Reply:
I'm not super familiar with the code base so the change is the PR should behave as the existing state of things. Removing the `decodeURIComponent` would deviate from that.

Reply:
Can you share the package lock file in a ticket? Happy to take a stab since there could be multiple places that might need fixing.

Reply:
I figured it out. In this version of `PackageURL` we need to encode the `@` when passing it to .fromString() with `encodeURIComponent`

Reply:
missed a `/`. fixed.